.pipelines/pipeline.yaml

```diff
@@ -331,7 +331,7 @@ stages:
       clusterName: "overlaye2e"
```

Comment:
nit: Should we refactor this clusterName to `ciliumoverlaye2e` to match its counterpart `ciliumoverlaycnie2e`